io-ts/__tests__/__snapshots__/io-ts.ts.snap

```diff
@@ -0,0 +1,59 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ioTsResolver should return a single error from ioTsResolver when validation fails 1`] = `
+Object {
+  "errors": Object {
+    "luckyNumbers": Array [
+      undefined,
+      undefined,
+      Object {
+        "message": "expected number but got \\"3\\"",
+        "ref": undefined,
+        "type": "number",
+      },
+    ],
+  },
+  "values": Object {},
+}
+`;
+
+exports[`ioTsResolver should return all the errors from ioTsResolver when validation fails with \`validateAllFieldCriteria\` set to true 1`] = `
```

Comment:
When `validateAllFieldCriteria` is set to `true`, the error object contains a property `types` with all the errors, here is an example: https://github.com/react-hook-form/resolvers/blob/master/zod/src/__tests__/__snapshots__/zod.ts.snap#L258

Reply:
oh how interesting ! I've used the simple single, way but it's a good point

Reply:
ok, it should be working as expected now